js/main2e.js

```diff
@@ -126,7 +122,7 @@ var TAPE_TYPES = ['wav','aiff','aif','mp3','m4a'];
 
 var _currentDrive = 1;
 
-function openLoad(drive, event)
+window.openLoad = function(drive, event)
```

Comment:
Are you meaning to set these on the global `window` object? If so, what's the advantage? Event listeners?

Reply:
It results in the smallest delta at the moment.  I want to clean this up into a class in the near future, but this got things moving.